examples/elasticsearch/es-rc.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: es
+  namespace: default
+  labels:
+    component: elasticsearch
+spec:
+  replicas: 1
+  selector:
+    component: elasticsearch
+  template:
+    metadata:
+      labels:
+        component: elasticsearch
+    spec:
+      serviceAccount: elasticsearch
+      containers:
+      - name: es
+        image: quay.io/pires/docker-elasticsearch-kubernetes:1.7.1-4
+        env:
+        - name: KUBERNETES_TRUST_CERT
+          value: "true"
```

Comment:
In fact, scratch the `KUBERNETES_SERVICE_HOST` recommendation - that was a legacy of a bad cert in a few of my environments that meant the cert didn't have the service IP as an IP SAN. Ignore :)

Just add the `KUBERNETES_CA_CERTIFICATE_FILE` env var & remove the `KUBERNETES_TRUST_CERT` env var so that we get proper secured comms to api server.